### PR TITLE
Open figures via the context menu

### LINKED ISSUE #156: Opening figures from vscode
**Is your feature request related to a problem? Please describe.**
Opening figures from the VS Code UI does not seem to be something that is currently supported. Opening the figure outside VS Code, in my experience, requires me to close VS Code so the MATLAB session is freed. A solution that does not require closing VS Code while leveraging the UI is desired.

**Describe the solution you'd like**
An option to open a figure from a context menu by right-clicking on a `.fig` file from the file explorer would be a nice solution, much like how adding files/folders to the MATLAB path works for the extension right now. The figure should open in a separate window.

**Describe alternatives you've considered**
I currently do the following to open a `.fig` file from VS Code:
1. Right-click the `.fig` file that I want to open
2. Select "Copy Path" from the context menu
3. Open the MATLAB terminal, call the `openfig` function, and paste the copied path in as an argument to the function

### PR DESCRIPTION
**Pull request by DFriend01**
Resolves #156. This PR implements opening figures from the context menu for `.fig` files only.

### CODE REVIEW

**Comment by DFriend01** [comment]
Sorry for the mess, squashing commits went wrong :-)

**Comment by dklilley** [comment]
Thank you for the pull request! I will take a look at it next week.

**Review by dklilley — CHANGES_REQUESTED** [review]
This looks good to me! Thank you for adding this!
Can you add an entry to the changelog about this feature?

**Comment by nothans** [comment]
Can you contact me about signing a CLA? Thank you.

**Comment by DFriend01** [comment]
@nothans I sent you an email

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Enabled users to open MATLAB .fig files through the context menu
```
Comment (dklilley) [inline CHANGELOG.md:11]: Can you move this under the existing `### Added` section?

**Comment by Gusmano-2-OSU** [comment]
Why wasn't this merged into main?

**Pull request closed without merging**